src/axom/sidre/datacollection/MFEMSidreDataCollection.hpp

```diff
@@ -0,0 +1,558 @@
+// Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#ifndef MFEM_SIDRE_DATACOLLECTION_HPP_
+#define MFEM_SIDRE_DATACOLLECTION_HPP_
+
+#include "axom/config.hpp"
+
+#ifdef AXOM_USE_MFEM
+
+#include "mfem.hpp"
+
+#include "axom/sidre/core/sidre.hpp"
+
+namespace axom
+{
+
+namespace sidre
+{
+
+/** @brief Data collection with Sidre routines following the Conduit mesh
+    blueprint specification. */
+/** MFEMSidreDataCollection provides an HDF5-based file format for visualization
+   or
+    restart capability.  This functionality is aimed primarily at customers of
+    LLNL's axom project that run problems at extreme scales.
+
+    For more information, see:
+    - Sidre component of LLNL's axom project (to be open-sourced),
```

Comment:
It has been open-sourced! man would we be in trouble otherwise... (yes i realize its a copy/paste.. i have to amuse myself)

Reply:
I'd say this comment should probably go away since this is now *part* of sidre... 😄